pages/storage.vue

```diff
@@ -143,7 +130,7 @@ export default {
   computed: {
     services: {
```

Comment:
I haven't used the getter/setter with computed much.  Does this still work if we change that computed to `services() => this.tool.services`?

Reply:
yeah i think it still works.